src/behavior/src/A32NX_Interior_Generics.xml

```diff
@@ -737,4 +737,31 @@
         </UseTemplate>
     </Template>
 
+
+    <!--
+    Template for toggling of light potentiometers that not are bound to a specific lightdef.Type.
+    Needs a condition, requisite and a potentiometer to toggle.
+
+    Main Parameters:
+            - SYNC_FREQUENCY        Default: 1          Speed of update loop, N per second.
+            - TOGGLE_CONDITION      <Boolean>           The condition to trigger a toggle.
+            - TOGGLE_REQUISITE      <Boolean>           The required powered BUS for potentiometer.
+    -->
+    <Template Name="FBW_Potentiometer_Toggle">
+        <DefaultTemplateParameters>
+            <SYNC_FREQUENCY>1</SYNC_FREQUENCY>
+            <TOGGLE_REQUISITE></TOGGLE_REQUISITE>
+            <TOGGLE_CONDITION></TOGGLE_CONDITION>
+        </DefaultTemplateParameters>
+
+        <UseTemplate Name="ASOBO_GT_Update">
+            <FREQUENCY>#SYNC_FREQUENCY#</FREQUENCY>
+            <UPDATE_CODE>
+                #TOGGLE_REQUISITE# if{
+                    #TOGGLE_CONDITION# 100 * #POTENTIOMETER# (&gt;K:2:LIGHT_POTENTIOMETER_SET)
+                }
```

Comment:
**issue**: When the `TOGGLE_CONDITION` is `1`, and `TOGGLE_REQUISITE` is also `1`, it shows the light. When `TOGGLE_REQUISITE` then becomes `0`, it will not stop showing the light.
**thought**: Probably `POTENTIOMETER` should be described as a parameter somewhere?

Reply:
@davidwalschots Logic updated + `POTENTIOMETER` described as parameter in template.